### ISSUE TITLE
script to install bcr robot

### ISSUE THREAD

**Issue by ncnynl** [blackcoffeerobotics/bcr_bot#43]
https://github.com/ncnynl/commands/blob/master/commands_extra/ros2_gazebo/shell/install_bcr_bot.sh

add a rapid script to install bcr bot for ros2 

```
curl -k https://www.ncnynl.com/rcm.sh | bash -
rcm -s install_bcr_bot
```

run:

1. ros2 launch bcr_bot ign.launch.py
2. ros2 launch bcr_bot rviz.launch.py

**Comment by D-1shu** [comment]
@ncnynl Thank you for the installation script for bcr_bot. We now have binaries available for easier installation. Could you try using them and let us know your feedback? 
```bash
sudo apt-get install ros-humble-bcr-bot
```
We may include your script in a future release.

Best regards,
Devanshu Sharma
Black Coffee Robotics

**Comment by ncnynl** [comment]
@D-1shu 

Yes, I understand your apt installation method and tested your code. Thank you for providing such a good program.

By learning your URDF, I fixed the point cloud problem of my rtabmap mapping. Based on learning needs, we still need to install the source code version, especially if we want to do secondary development.

So I wrote a script and also wrote a concise tutorial on how to quickly install and use bcr bot,
https://www.ncnynl.com/archives/202407/6453.html
